app/assets/javascripts/components/school_search.js.erb

```diff
@@ -83,14 +83,15 @@ document.addEventListener("DOMContentLoaded", function() {
         var school = findSchool(id);
 
         var suggestion =
-          '<label class="govuk-label govuk-label--s">' +
-          school.name +
-          "</label>" +
-          '<span class="govuk-hint">' +
-          school.address +
-          "</span>";
-
-        if (school.closeDate) suggestion += '<span class="govuk-hint">' + "Closed on " + school.closeDate + "</span>";
+          '<div class="flex"><div class="flex-1"><label class="govuk-label govuk-label--s">' + school.name + "</label>";
+        suggestion += '<span class="govuk-hint">' + school.address + "</span></div>";
```

Comment:
If we go for 1 line per tag (separated by `+`s), it makes it easier to see the structure here. I think we're missing a closing `</div>` if we don't have a close date.